### PR TITLE
Fix ical import

### PR DESCRIPTION
**Pull request by ChristophWurst**
* Fix limited content type filter
* Fix Axios sending invalid requests

Thanks @georgehrke!

### CODE REVIEW

File: lib/Controller/MessagesController.php [hunk 1]
```diff
@@ -443,7 +443,7 @@ private function attachmentIsImage(array $attachment): bool {
 	 * @return boolean
 	 */
 	private function attachmentIsCalendarEvent(array $attachment): bool {
-		return $attachment['mime'] === 'text/calendar';
+		return in_array($attachment['mime'],  ['text/calendar', 'application/ics'], true);
```
Comment (georgehrke) [inline lib/Controller/MessagesController.php]: Why `'application/ics'`?

Comment (ChristophWurst) [reply lib/Controller/MessagesController.php]: That's the mime set for some emails apparently

File: src/service/DAVService.js [hunk 2]
```diff
@@ -206,15 +204,11 @@ export const importCalendarEvent = url => data => {
...
-					Axios.put(
-						url + getRandomString(),
-						{data},
-						{
-							headers: {
-								contentType: 'text/calendar; charset=utf-8',
-							},
-						}
-					)
+					Axios.put(url + getRandomString(), component, {
```
Comment (georgehrke) [inline src/service/DAVService.js]: Please add '.ics' to the filename :)

File: lib/Controller/MessagesController.php (hunk 1 above)
Comment (ChristophWurst) [reply lib/Controller/MessagesController.php]:
```
Content-Type: application/ics; name=meetup.ics; method=PUBLISH; 
        charset=UTF-8
Content-Transfer-Encoding: base64
Content-Disposition: attachment; filename=meetup.ics
```

Comment (ChristophWurst) [reply lib/Controller/MessagesController.php]: https://gitlab.freedesktop.org/xdg/shared-mime-info/commit/9d73fc74d9386333de1ba49ffe7ba991ef498c37

I think this is fine :)

**Pull request merged**